### PR TITLE
Weekly thread #5-#7, use actual created_at timestamp in Thread creation

### PR DESCRIPTION
**Pull request by lukinovec**

### CODE REVIEW

**Comment by stancl** [comment]
Which week should it be? The commit says #5, files say #7, but I think it's week 6

**Comment by lukinovec** [comment]
> Which week should it be? The commit says #5, files say #7, but I think it's week 6

Yeah, you're right. It should be week 6.

File: content/threads/weekly-thread-2022-05.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+slug: weekly-thread-2022-05
+title: 'Weekly thread #05 of 2022'
+tweet_id: '1489620159585153026'
+author_username: archtechx
+created_at: 2022-02-17T16:16:06+00:00
```
Comment (stancl) [inline content/threads/weekly-thread-2022-05.md:6]: The threads are both on 17/2, but the date should be the same as on Twitter

Comment (stancl) [reply content/threads/weekly-thread-2022-05.md:6]: Also, all the tweets seem to be either on 4/2 or 16/2? I think I asked about this via chat

Comment (lukinovec) [reply content/threads/weekly-thread-2022-05.md:6]: > The threads are both on 17/2, but the date should be the same as on Twitter

I see, I'll fix it.

> Also, all the tweets seem to be either on 4/2 or 16/2? I think I asked about this via chat

Yeah, the #5 thread was posted on 4/2 and #6 was posted on 16/2. I think the tweet `created_at` timestamps are correct (but the thread's `created_at` is not).

**Comment by stancl** [comment]
can you add the last thread here as well?

**Comment by stancl** [comment]
```
16:18:36.910 | > php artisan orbit:clear && php artisan html:generate static http://127.0.0.1:7777
-- | --
16:18:37.508 |  
16:18:37.509 | Facade\Ignition\Exceptions\ViewException
16:18:37.509 |  
16:18:37.509 | Trying to get property 'profile_url' of non-object (View: /opt/buildhome/repo/resources/views/components/author-card.blade.php)
16:18:37.510 |  
16:18:37.510 | at resources/views/components/author-card.blade.php:7
16:18:37.510 | 3▕     'links' => [],
16:18:37.510 | 4▕ ])
16:18:37.510 | 5▕
16:18:37.510 | 6▕ <div class="flex w-full p-6 space-x-6 bg-gray-50 rounded-2xl">
16:18:37.510 | ➜   7▕     <a href="{{ $tip->author->profile_url }}" target="_blank">
16:18:37.510 | 8▕         <img class="bg-gray-200 rounded-full w-14 h-14" src="{{ asset($tip->author->avatar) }}"
16:18:37.510 | 9▕             alt="{{ $tip->author->name }}">
16:18:37.510 | 10▕     </a>
16:18:37.510 | 11▕
16:18:37.512 |  
16:18:37.512 | 1   resources/views/components/author-card.blade.php:7
16:18:37.512 | Illuminate\Foundation\Bootstrap\HandleExceptions::handleError()
16:18:37.512 |  
16:18:37.512 | 2   resources/views/tips/show.blade.php:105
16:18:37.512 | Illuminate\View\Factory::renderComponent()
16:18:37.519 | Script php artisan orbit:clear && php artisan html:generate static http://127.0.0.1:7777 handling the generate event returned with error code 1
16:18:37.529 | Failed: build command exited with code: 1
```

**Comment by lukinovec** [comment]
> ```
> 16:18:36.910 | > php artisan orbit:clear && php artisan html:generate static http://127.0.0.1:7777
> -- | --
> 16:18:37.508 |  
> 16:18:37.509 | Facade\Ignition\Exceptions\ViewException
> 16:18:37.509 |  
> 16:18:37.509 | Trying to get property 'profile_url' of non-object (View: /opt/buildhome/repo/resources/views/components/author-card.blade.php)
> 16:18:37.510 |  
> 16:18:37.510 | at resources/views/components/author-card.blade.php:7
> 16:18:37.510 | 3▕     'links' => [],
> 16:18:37.510 | 4▕ ])
> 16:18:37.510 | 5▕
> 16:18:37.510 | 6▕ <div class="flex w-full p-6 space-x-6 bg-gray-50 rounded-2xl">
> 16:18:37.510 | ➜   7▕     <a href="{{ $tip->author->profile_url }}" target="_blank">
> 16:18:37.510 | 8▕         author->avatar) }}"
> 16:18:37.510 | 9▕             alt="{{ $tip->author->name }}">
> 16:18:37.510 | 10▕     </a>
> 16:18:37.510 | 11▕
> 16:18:37.512 |  
> 16:18:37.512 | 1   resources/views/components/author-card.blade.php:7
> 16:18:37.512 | Illuminate\Foundation\Bootstrap\HandleExceptions::handleError()
> 16:18:37.512 |  
> 16:18:37.512 | 2   resources/views/tips/show.blade.php:105
> 16:18:37.512 | Illuminate\View\Factory::renderComponent()
> 16:18:37.519 | Script php artisan orbit:clear && php artisan html:generate static http://127.0.0.1:7777 handling the generate event returned with error code 1
> 16:18:37.529 | Failed: build command exited with code: 1
> ```

Fixed. Looks OK in the preview now.

**Comment by stancl** [comment]
Thanks!

**Pull request merged**